.nancy-ignore

```diff
@@ -4,24 +4,24 @@
 # Taken from: https://github.com/giantswarm/opsctl/pull/1072/files#diff-bbe4a7fb12c43622bce7c6840c770e9995be614626a219942ca138403629cb69R1
 CVE-2019-10743 until=2021-10-17
 
-CVE-2018-1714 until=2022-06-30
-CVE-2022-21221 until=2022-06-30
-CVE-2022-29162 until=2022-06-30
-CVE-2021-43784 until=2022-06-30
-CVE-2022-28357 until=2022-06-30
-CVE-2022-26652 until=2022-06-30
-CVE-2022-29946 until=2022-06-30
-CVE-2022-24450 until=2022-06-30
-CVE-2021-3127 until=2022-06-30
-CVE-2021-42576 until=2022-06-30
-CVE-2021-23772 until=2022-06-30
-CVE-2022-24687 until=2022-06-30
-CVE-2022-29153 until=2022-06-30
-CVE-2020-28483 until=2022-06-30
-CVE-2022-24778 until=2022-06-30
-CVE-2022-23648 until=2022-06-30
+CVE-2018-1714 until=2022-07-30
```

Comment:
Same as https://github.com/giantswarm/chart-operator/pull/905. Will rebase if that gets merged before this is approved.